docs/api/scheduler.md

```diff
@@ -854,7 +854,7 @@ Fetch all users usage given a Partition and displays general information about t
 
 ## UserTracker
 ### Get specific user usage tracking information
-Fetch specific user usage given a Partition and displays general information about the users managed by YuniKorn.
+Fetch specific user usage given a Partition and displays general information about the users managed by YuniKorn. In case username contains any special characters, needs to be url escaped to avoid issues.
```

Comment:
Nit: "In case the queue name contains any special characters, it needs to be url escaped to avoid issues."